sdk/src/main/scala/io/horizen/account/state/events/AddMcAddrOwnership.scala

```diff
@@ -0,0 +1,29 @@
+package io.horizen.account.state.events
+
+import io.horizen.account.state.events.annotation.{Indexed, Parameter}
+import io.horizen.evm.Address
+import org.web3j.abi.datatypes.generated.{Bytes3, Bytes32}
+import org.web3j.abi.datatypes.{Address => AbiAddress}
+
+import java.nio.charset.StandardCharsets
+import java.util
+import scala.annotation.meta.getter
+
+case class AddMcAddrOwnership(
+   @(Parameter @getter)(1) @(Indexed @getter) scAddress: AbiAddress,
+   @(Parameter @getter)(2) mcAddress_3: Bytes3,
+   @(Parameter @getter)(3) mcAddress_32: Bytes32
+) {}
+
+
+object AddMcAddrOwnership {
```

Comment:
The id is the keccak hash of the concatenation of addresses, a user could easily compute it if needed

Reply:
Agree that it can be computed easily but from a client perspective it would be easier compare a string instead of doing the keccak hash. If it has no cost for us why don't do it?

Reply:
Correct, the native smart contract consumes gas when adding logs, depending on event data size.